plans/subplans/install.pp

```diff
@@ -324,23 +324,28 @@
     ]
 
     $role_and_group =
```

Comment:
Change request:

If we're switching to csr_attributes.yaml pre-population for all servers, let's strip it out of this block entirely (remove `$role_and_group` and anything that operates on it), and instead add the needed `peadm::util::insert_csr_extension_requests` task run to `$csr_yaml_jobs` on [line 215](https://github.com/puppetlabs/puppetlabs-peadm/blob/7b21708d596dcb5f003203b2b445f55ffa9adf1e/plans/subplans/install.pp#L215-L240).

Reply:
Added a commit with the requested change.